wire-ios-sync-engine/Source/UserSession/OperationStatus.swift

```diff
@@ -86,8 +114,15 @@ public class OperationStatus: NSObject {
         }
     }
 
+    var hasPendingCall = false {
+        didSet {
+            updateOperationState()
+        }
+    }
+
     public var hasOngoingCall = false {
         didSet {
+            hasPendingCall = false
```

Comment:
Is this safe to set this to false each time `hasOngoingCall` is set ?

Reply:
I think so. My reasoning was that if we say a call is ongoing, or it is no longer ongoing, then we no longer have a pending call.

Reply:
I can think of one scenario where this doesn't work. 
- User A has 1:1 call with user B
- User C calls users A & B in a group call
- User A answers the incoming call
-> The call between user A & B gets terminated > `hasOngoingCall` is set to false > `hasPendingCall` is also set to false 

User B still hasn't answered the call but cannot because there is no pending call according to the operation status